ql/src/test/queries/clientpositive/authorization_privilege_objects.q

```diff
@@ -0,0 +1,20 @@
+--! qt:authorizer
+set hive.security.authorization.manager=org.apache.hadoop.hive.ql.security.authorization.plugin.sqlstd.SQLStdHiveAuthorizerFactoryForTest;
+set test.hive.authz.sstd.validator.outputPrivObjs=true;
+set hive.test.authz.sstd.hs2.mode=true;
+set user.name=testuser;
+
+CREATE DATABASE test_db;
+CREATE TABLE test_privs(i int);
+set user.name=testuser2;
+CREATE TABLE test_privs2(s string, i int);
+set user.name=testuser;
+SHOW DATABASES;
```

Comment:
I need to make changes here, because concurrent tests can interfere (I.E. they can create tables and databases that show up in the output).